docs/nginx.md

```diff
@@ -3,4 +3,46 @@ id: nginx
 title: nginx config
 ---
 
-todo
+You may want to use your nginx server as a reverse proxy to run gotify.
+
+Here is a sample config file if you run your gotify instance on port 1245
+
+```nginx
+upstream gotify {
+  # Set the port to the one you are using in gotify
+  server 127.0.0.1:1245;
+}
+
+server {
+  listen 80;
+
+  # Here goes your domain / subdomain
+  server_name push.example.com;
+
+  # Error logs, change this to wherever you installed gotify
```

Comment:
Could we remove these log config, similar to the apache config it should only have the minimum required config.